docs/src/pages/company/careers/developer-advocate.md

```diff
@@ -31,11 +41,27 @@ We are looking for a dedicated educator and experienced developer to join us as
 
 ### What you'll do on a day-to-day basis
 
-- You will compose blog posts. You will be writing tutorials and other technical blog posts.
-- You will participate in and speak at events, and online live-streams.
+- You will monitor success metrics, consider how effective initiatives are at achieving goals, and adjust effort accordingly.
+- You will create and implement plans to build, enable and nurture the community.
+- You will compose interesting blog posts. You will be writing tutorials and other technical content.
+- You will participate in and speak at events, and online live streams.
+- You will build inspiring apps, examples, and improve the documentation where needed.
 - You will create technical videos, video tutorials, demos, and additional videos for the MUI YouTube channel.
-- You will build apps, examples, and improve the documentation where needed.
 - You will give feedback to product management, to influence the product roadmaps based on developers' needs.
+- You will make sure contributions to the documentation are of high quality, sharing best practices with the developers.
+- You will help other team members grow at engaging with the community. For instance by encouraging and providing critical feedback on new blog posts.
+- You will rework the pages of the documentation that are confusing, base on feedback.
+
```

Comment:
```suggestion
- You will follow GitHub issues to understand where developers face frustration, and develop strategies to overcome these. This could be suggesting or implementing documentation updates, or proposing or contributing code changes that solve the core issue.
```